lms/services/lti_user.py

```diff
@@ -19,20 +19,22 @@ def __init__(
         self._lti_roles_service = lti_role_service
         self._application_instance_service = application_instance_service
 
-    def from_auth_params(self, application_instance, lti_core_schema) -> LTIUser:
-        """Create an LTIUser from a LTIV11CoreSchema like dict."""
+    def from_lti_params(self, application_instance, lti_params) -> LTIUser:
```

Comment:
Accessing the whole LTIParams here now to build LTIUser.

IMO a bigger refactor should tackle splinting authentication and LTIUser generation but I want to keep this focused.